CONTRIBUTING.md

```diff
@@ -36,6 +36,217 @@ sudo ifconfig lo0 alias 127.0.0.3 up
 sudo ifconfig lo0 alias 127.0.0.4 up
 ```
 
+## Testing in Crawlee with vitest
+
+There are a few small differences between how testing in jest and vitest works. Mostly, they relate to what to do, and not do anymore.
+
+### Configuration file for tests created in the package they are for
+
+You will need to use this tsconfig.json in the `test` folder in the package (say, if you were adding a test to `packages/core` and there wasn't a `tsconfig.json` file already there)
+
+```json
+{
```

Comment:
all the code snippets in the file are wrongly indented

Reply:
Times like these I wish we had prettier... will fix